region/maridia/inner-yellow/Plasma Spark Room.json

```diff
@@ -323,7 +323,8 @@
               "Charge+Wave",
               "Charge+Spazer"
             ]
-          }}
+          }},
+          "h_pauseAbuseMinimalReserveRefill"
```

Comment:
It may be that `{shinespark}` has to be ordered after this.  
It would have been somewhat wrong listing the shinespark before enemyDamage as it was.

Reply:
Good catch. I bet there are a lot of these problems in the project. I wonder if it would be possible to search for all strats that take damage and spark - then i could look through and verify them.